libpod/container_internal_linux.go

```diff
@@ -1247,6 +1275,79 @@ func (c *Container) restore(ctx context.Context, options ContainerCheckpointOpti
 		}
 	}
 
+	if options.Pod != "" {
```

Comment:
We need to make sure that the container config for the restored container has the ID of the new pod in it.

Reply:
I think I solved also this.